src/style.css

```diff
@@ -0,0 +1,8 @@
+body {
+  margin: 0;
+}
+
+.stars__block {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
This class is not used and not needed (you have stars-block not stars__block in html anyways)
```suggestion
```